### ISSUE TITLE
Android compatible?

### ISSUE THREAD

**Issue by alessdiimperio** [apple/swift-foundation#359]
If the Swift-stdlib can run on android and swift foundation is being built purely in swift, does this mean that swift-foundation will also run on android?

Would be amazing to have a KMM equivalent using Swift for cross platform development with shared business logic.

**Comment by iCharlesHu** [comment]
Hi @alessdiimperio yes! Our plan is to make `SwiftFoundation` run everywhere that Swift runs. We certainly welcome contributions in that effort. Feel free to create PRs or file issues for specific problems you are seeing.

**Comment by finagolfin** [comment]
> Would be amazing to have a KMP equivalent using Swift

I don't know what KMP is, but people are [already running the current swift-corelibs-foundation on Android](https://github.com/finagolfin/swift-android-sdk). Try building Swift packages with that SDK, and let me know there if you have any problems.

**Comment by alessdiimperio** [comment]
KMP = kotlin multiplatform and allows developers to share code between android and ios apps using kotlin code.

https://kotlinlang.org/docs/multiplatform.html#sample-projects

Is there any good source of documentation for running swift code on android and how to use platform specific api's easily?
As well as interaction between swift and compose.

**Comment by finagolfin** [comment]
> Is there any good source of documentation for running swift code on android and how to use platform specific api's easily?

No, not that I know of. Android is still a community-supported platform, so [the pieces exist in various places](https://github.com/SwiftAndroid/swift-jni/tree/devel), but you still have to put them together yourself.
